### PR TITLE
FR10.2 Implementing and comitting

### PR DESCRIPTION
**Pull request by Rubberduck1**
@Jabul99 could you have a look at it ;-).

### CODE REVIEW

File: test/NewBankTest.java [hunk 1]
```diff
@@ -438,5 +438,36 @@ public void getHelp() {
...
+        Assertions.assertEquals("Success. Loan of 500.0 offered from 55555888 for 365 days with interest of 5%", result);
+        Assertions.assertEquals(500, account.getAvailableBalance());
+
+        String result2 = test.processRequest(clientId2, "ACCEPTLOAN 1 55555999");
+
+        Assertions.assertEquals("Success. Loan number 1 accepted by account 55555999.", result2);
+
+        String result3 = test.processRequest(clientId, "PAYBACKLOAN 1");
+
+        Assertions.assertEquals("Success. Loan Number: 1, Account Number From: 55555999, Account Number To: 55555888, Amount: 525.0\n", result3);
```
Comment (Jabul99) [inline test/NewBankTest.java:469]: This test does not check whether the individual bookings on the accounts have been made correctly. This test should be added.

File: test/NewBankTest.java [hunk 2]
```diff
@@ -438,5 +438,36 @@ public void getHelp() {
                 "PAY <Amount> <Debit account> <Credit account>\tPays funds from one account to another account, which may be held by another customer\n" +
                 "HELP\t\t\tShows this menu\n\n", result);
     }
```
Comment (Jabul99) [inline test/NewBankTest.java:440]: The negative cases should also be tested

File: test/NewBankTest.java (hunk 1 above)
Comment (Rubberduck1) [reply test/NewBankTest.java:469]: Thank you very much for the good advice. I will add to it.

File: test/NewBankTest.java (hunk 2 above)
Comment (Rubberduck1) [reply test/NewBankTest.java:440]: Thank you. Will be optimised ;-).

File: test/NewBankTest.java (hunk 1 above)
Comment (Jabul99) [reply test/NewBankTest.java:469]: Nicely done

File: newbank/server/NewBank.java [hunk 3]
```diff
@@ -177,6 +177,7 @@ private String paybackLoan(CustomerID customerID, int loanNumber) {
                 amount += loan.getAmount() / 100 * loan.getInterest();
                 accountFrom.debit(amount);
```
Comment (Jabul99) [inline newbank/server/NewBank.java:178]: The balance of the account should be tested for sufficient funds before a transfer is made.

Comment (Rubberduck1) [reply newbank/server/NewBank.java:178]: Thank you, I will add that.

Comment (Jabul99) [reply newbank/server/NewBank.java:181]: Nicely done

Comment (Rubberduck1) [reply newbank/server/NewBank.java:181]: The account balance is now checked for sufficient funds.

**Comment by Rubberduck1** [comment]
Implemented Sub requirements

FR10.2.1 The system should accept the command PAYBACKLOAN <LoanNumber>
FR10.2.2 The system should check the validity of the loan number
FR10.2.3 The system should pay back the amount to the customer with interest if the validity check was fulfilled
FR10.2.4 The system should display a success message if the payback of a micro-loan was successful.
FR10.2.5 The system should display a failure message if the payback of a micro-loan was not successful.

**Pull request merged**